src/main/java/com/transportcompany/transportcompany/controllers/UserController.java

```diff
@@ -1,46 +1,43 @@
 package com.transportcompany.transportcompany.controllers;
 
 import com.transportcompany.transportcompany.models.dtos.UserDTO;
-import com.transportcompany.transportcompany.models.entities.User;
-import com.transportcompany.transportcompany.repositories.UserRepository;
 import com.transportcompany.transportcompany.services.UserService;
-import jakarta.validation.Valid;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
-import java.util.Optional;
-
 
 @RestController
 public class UserController {
 
     @Autowired
```

Comment:
We don't use `@Autowired` annotation on fields when we are doing the construction injection. In our case, we can use this annotation before line 17, on the constructor. At team level, please read about what @Autowired does and what constructor injection means (regarding the dependencies).